harvester_e2e_tests/integration/test_hosts.py

```diff
@@ -303,40 +304,40 @@ def test_poweroff_node_trigger_vm_migrate(
             f", script path: {host_state.path}"
         )
 
-    # check vm is migrated to another host
-    endtime = datetime.now() + timedelta(seconds=wait_timeout)
-    while endtime > datetime.now():
-        code, data = api_client.vms.get_status(focal_vm['name'], focal_vm['namespace'])
-        if data.get('status', {}).get('phase', "") == "Running":
-            assert src_host != data['status']['nodeName'], (
-                f"Failed to migrate vm {focal_vm['namespace']}/{focal_vm['name']} \
-                    from {src_host} to another node"
+    try:
+        # XXX: default `node-monitor-grace-period` is 5 minutes
+        # ref: https://github.com/harvester/harvester/issues/3896#issuecomment-1553154738
+        sleep(5 * 60 + vm_force_reset_policy['period'])
+        # check vm is restarted
```

Comment:
Has the VM restarted on another host?

Reply:
We don't check the VM will be restarted on another host but restarted (rescheduled) because it could still be scheduled on the same host and started successfully. (which depends on Kubevirt's scheduling rules)

Reply:
>because it could still be scheduled on the same host and started successfully

If a host is powered off, how can a vm restart on it?